### PR TITLE
2021-07-07-release-20217.markdown: fix typos

### PR DESCRIPTION
**Pull request by RubenKelevra**
## Proposed change
Spelling, grammar or other readability improvements

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2021-07-07-release-20217.markdown [hunk 1]
```diff
@@ -19,12 +19,12 @@ feedback: true
 Happy July, which means Home Assistant Core 2021.7!
 
 An interesting release, with a bunch of little goodies to make things easier in
-creating automations, scripts and doing templating. Those are things that in
+creating automations, scripts, and doing templating. Those are things that in
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: We avoid using the oxford commas in all our communication and documentation.
This is documented in our style guides.

```suggestion
creating automations, scripts and doing templating. Those are things that in
```

File: source/_posts/2021-07-07-release-20217.markdown [hunk 2]
```diff
@@ -56,8 +56,7 @@ Alright, that's it! Enjoy the release!
 
 ## New entity: Select
 
-In this release we welcome the `select` entity to the Home Assistant family. The
-select entity is family of the dropdown helper (also known as
+In this release, we welcome the `select` entity to the Home Assistant family. The select entity is a close relative of the dropdown helper (also known as
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: "relative" was removed because of feedback, as "relative" was interpreted wrongly by some non-native speakers. Hence it was replaced by "family" during the beta.

File: source/_posts/2021-07-07-release-20217.markdown [hunk 3]
```diff
@@ -80,10 +79,9 @@ Genie you can now change the room size for your diffuser.
 ## Trigger conditions and trigger IDs
 
 If you are creating some complex automations in YAML, you might be familiar with
-this. Consider an big automation, with a whole bunch of triggers. But how
-would you know which of those triggers actually triggered the automation?
+this. Consider a big automation, with a whole bunch of triggers. But how would you know which of those triggers actually triggered the automation?
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: I don't see a change in this line, besides removing a new line that has no effect?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 4]
```diff
@@ -80,10 +79,9 @@ Genie you can now change the room size for your diffuser.
...
 
-You can now assign an `id` to your triggers that is passed into automation when
+You can now assign an `id` to your triggers that are passed into automation when
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: It is referring to `id` which is singular, thus "is" and not "are".

File: source/_posts/2021-07-07-release-20217.markdown [hunk 5]
```diff
@@ -138,8 +136,7 @@ Screenshot of using a trigger condition in the automation editor.
 ## Script debugging
 
 In [Home Assistant Core 2021.4](/blog/2021/04/07/release-20214/#automation-debugging),
-we added the ability to debug automations. This release, we've made these
-same powerful tools available for scripts!
+we added the ability to debug automations. In this release, we've made these same powerful tools available for scripts!
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: ?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 6]
```diff
@@ -160,8 +157,7 @@ A small, but possibly helpful, change to our script and automations.
 You can now reference other entities for the above/below values of numeric
 state triggers and conditions. Both sensors and number entities can be used.
 
-For example, you can now trigger an automation if the outside temperature
-is higher than the temperature inside.
+For example, you can now trigger an automation if the outside temperature is higher than the temperature inside.
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: No change?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 7]
```diff
@@ -189,8 +185,8 @@ is hard. And honestly, that will never go away, times, dates and timezones are
 complex little beasts.
 
 However, we realized that the hardest part of using date & times with templates
-is converting the state of a sensor or text to a datetime. This
-release we added a small template method to help with that: `as_datetime`.
+is converting the state of a sensor or text to a datetime. In this
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: ?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 8]
```diff
@@ -212,7 +208,7 @@ of Home Assistant.
 
 Thanks to [@kmdm], as of this release, we also provide a series version tag
 that always points to the latest patch version of that release, in addition
-to all existing tags we already provide.
+to all existing tags, we already provide.
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: This comma is actually incorrect, please revert this one.

File: source/_posts/2021-07-07-release-20217.markdown [hunk 9]
```diff
@@ -227,26 +223,21 @@ There is much more juice in this release; here are some of the other
 noteworthy changes this release:
 
 - Z-Wave JS got quite a few updates this release:
-  - A new `zwave_js.multicast_set_value` is available, allowing to issue
-    a set value command via multicast. Thanks, [@raman325]!
-  - Each node now has a status sensor available and can be pinged using the
-    new `zwave_js.ping` service. Added by [@raman325].
+  - A new `zwave_js.multicast_set_value` is available, allowing to issue a set value command via multicast. Thanks, [@raman325]!
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: I don't see a change in this line?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 10]
```diff
@@ -227,26 +223,21 @@ There is much more juice in this release; here are some of the other
...
+  - Each node now has a status sensor available and can be pinged using the new `zwave_js.ping` service. Added by [@raman325].
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: I don't see a change in this line? Why was the new line removed?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 11]
```diff
@@ -227,26 +223,21 @@ There is much more juice in this release; here are some of the other
...
   - The Z-Wave JS configuration panel now has a "Heal Network" button,
     thanks [@cgarwood]!
   - Z-Wave JS Server connection can now be re-configured from the Z-Wave JS
     configuration panel, added by [@MartinHjelmare].
   - Z-Wave JS logs can now be downloaded, thanks [@raman325]!
-- The Google Assistant integration now has support for fan speed percentages and
-  preset modes. Thanks, [@jbouwh]!
+- The Google Assistant integration now has support for fan speed percentages and preset modes. Thanks, [@jbouwh]!
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: I don't see a change in this line, why is the new line removed?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 12]
```diff
@@ -227,26 +223,21 @@ There is much more juice in this release; here are some of the other
...
 - [@jbouwh] didn't stop there and added fan preset mode support to Alexa too!
 - The Philips TV integration now supports Ambilights, added by [@elupus].
 - Yamaha MusicCast integration now supports grouping services, thanks [@micha91]!
 - [@raman325] added a whole bunch of sensors to the ClimaCell integration!
 - WLED now supports local push. Updates are now instantly both ways. Also,
   the master light can be kept and added support for controlling user presets.
-- Setting up Xiaomi devices has gotten way easier! There is no need to do
-  difficult things to get the tokens. Instead, Home Assistant can now extract
-  the tokens from a Xiaomi Cloud account. Thanks, [@starkillerOG]!
+- Setting up Xiaomi devices has gotten way easier! There is no need to do difficult things to get the tokens. Instead, Home Assistant can now extract the tokens from a Xiaomi Cloud account. Thanks, [@starkillerOG]!
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: Same as above? What is changed here?

File: source/_posts/2021-07-07-release-20217.markdown [hunk 13]
```diff
@@ -257,8 +248,7 @@ noteworthy changes this release:
 - If you have a SmartTube, you can now reset your reminders, thanks [@mdz]!
 - KNX had quite a lot of updates and added support for XY-color lights,
   thanks [@farmio].
-- [@OttoWinter] added support for presets, custom presets and custom fan modes
-  for climate controls in ESPHome. Awesome!
+- [@OttoWinter] added support for presets, custom presets, and custom fan modes for climate controls in ESPHome. Awesome!
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: This is adding an oxford comma, which we don't use. Please revert this change.

File: source/_posts/2021-07-07-release-20217.markdown [hunk 14]
```diff
@@ -337,16 +327,16 @@ rejected if the request is marked as forwarded.
...
 {% details "Python 3.9 / Alpine 3.13" %}
 
-Our Docker images are now based on Alpine 3.13, and run Python 3.9.
+Our Docker images are now based on Alpine 3.13 and run Python 3.9.
 
 This is mainly interesting if you running custom Docker containers based
 on our container.
 
-If you are using Home Assistant Container, Home Assistant OS or the Home Assistant
+If you are using Home Assistant Container, Home Assistant OS, or the Home Assistant
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: Please revert this comma addition.

File: source/_posts/2021-07-07-release-20217.markdown [hunk 15]
```diff
@@ -455,11 +443,7 @@ If you are looking for a replacement, you can use the
   `websocket-client` library which is incompatible with the requirements of
   other integrations that are actively maintained.
 
-- It's not clear if this integration still works with the gpmdp app that now
-  only supports YouTube Music. If there's someone that uses the integration
-  successfully and wants to take on the maintenance task that is required to get
-  the integration in a compatible state, please create an issue to discus
-   the future of this integration.
+- It's not clear if this integration still works with the gpmdp app that now only supports YouTube Music. If there's someone that uses the integration successfully and wants to take on the maintenance task that is required to get the integration in a compatible state, please create an issue to discuss the future of this integration.
```
Comment (frenck) [inline source/_posts/2021-07-07-release-20217.markdown]: Could you please keep the new lines in place, it is hard to find the change in this one now.

**Review by frenck — CHANGES_REQUESTED** [review]
Please, don't remove the new lines, it is really unneeded and making reviewing this change almost impossible.

Thanks! 👍

File: source/_posts/2021-07-07-release-20217.markdown (hunk 2 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: I'm a non-native speaker and couldn't find this usage in a dictionary. That's why I changed it. Relative itself might be ambiguous but "close relative" should be more easily understood than the uncommon usage of "family".

File: source/_posts/2021-07-07-release-20217.markdown (hunk 3 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: Yeah sorry, I think I made changes and reverted them again in some blocks, that's why I changed the new lines.

I'll revert that.

File: source/_posts/2021-07-07-release-20217.markdown (hunk 4 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: right, sorry.

File: source/_posts/2021-07-07-release-20217.markdown (hunk 8 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: ok

File: source/_posts/2021-07-07-release-20217.markdown (hunk 14 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: ok

File: source/_posts/2021-07-07-release-20217.markdown (hunk 13 above)
Comment (RubenKelevra) [reply source/_posts/2021-07-07-release-20217.markdown]: ok

**Comment by RubenKelevra** [comment]
I addressed the comments, should be all resolved except for the L56.

Hope this works out now, getting late here.

**Review by frenck — APPROVED** [review]
Awesome! Thanks, @RubenKelevra 👍
